core/src/main/scala/org/apache/spark/TaskState.scala

```diff
@@ -49,5 +49,6 @@ private[spark] object TaskState extends Enumeration {
     case MesosTaskState.TASK_KILLED => KILLED
     case MesosTaskState.TASK_LOST => LOST
     case MesosTaskState.TASK_ERROR => LOST
+    case MesosTaskState.TASK_KILLING => RUNNING
```

Comment:
Is it maybe clearer to use `case x | y | z => foo` syntax here to show clearly which several cases map to one output state? here and above. Either that, or at least loosely group the inputs; KILLING seems like it should be next to KILLED